### PR TITLE
fix some a11y color contrast issues

### LINKED ISSUE #3817: fix some color contrast issues
## overview
fix some a11y issues across automate related to color contrast. these were found using the [wave chrome extension](https://chrome.google.com/webstore/detail/wave-evaluation-tool/jbbplnpkjmmeebjpijfedlgcdilocofh?hl=en-US).

## details
- [ ] event feed summary labels need to be chef-primary-dark instead of chef-dark-gray since they are on a gray background
- [ ] big filter button labels across automate need to be chef-primary-dark instead of chef-dark-gray since they are on a gray background
- [ ] status colors across automate need to be updated to use accessible colors
- [ ] apps icons in right-panel need to use the same color as the text
- [ ] purple color on event feed is not accessible, update to using the purple disconnected color from apps
- [ ] control status when its 0 is not accessible @jonong1972 will inform how to fix

### PR DESCRIPTION
**Pull request by susanev**
### :nut_and_bolt: Description: What code changed, and why?
weve got a number of inaccessible colors across automate. this is the start to fixing them.

~im gonna need to get a ux team review on this because it darkens our colors across automate quite a bit.~

~im waiting on @jonong1972 before fixing the control status color issue, so leaving this in draft. but feel free to review now.~

#### update
- ux team has decided to fix only some a11y color issues, specifically leaving (1) compliance control zero statuses (2) failed status color (3) warning status color (4) okay status color. these will be fixed in a later pr after more time for alignment
- this pr should fix the rest of the a11y color issues not mentioned in the first item
- making chef-dark-gray slightly darker so its accessible on the gray page backgrounds
- making the chef-green darker so its accessible, only used in client run diffs
- attributes color now uses our chef blue link color that is accessible
- fixes issue on apps right panel that was not using the same color for the icons that was used for the text
- makes the client runs top checkbox accessible

### :chains: Related Resources
fixes #3817 

### :+1: Definition of Done
colors are accessible

### :athletic_shoe: How to Build and Test the Change
`make update-ui-lib` from `components/automate-ui`
`rebuild components/automate-ui-devproxy`

### :white_check_mark: Checklist

- [x] I have read the [CONTRIBUTING document](https://github.com/chef/automate/blob/master/CONTRIBUTING.md).
- [ ] Tests added/updated?
- [ ] Docs added/updated?
- [x] All commits have been signed-off for the [Developer Certification of Origin](https://github.com/chef/chef/blob/master/CONTRIBUTING.md#developer-certification-of-origin-dco).

##### event feed
event-feed-before

##### apps
apps-success-before
apps-warning-before
apps-fail-before

##### client-runs
client-runs-before
client-run-attributes-before
client-run-diff-before
client-run-resources-before

##### comliance
compliance-overview-before

##### event feed
event-feed

##### apps
apps-right-panel-warning
apps-right-panel-ok
apps-right-panel-fail

##### client-runs
client-run-resources
client-runs-diff
client-run-attributes
client-run-list

##### comliance
compliance-overview

### CODE REVIEW

**Comment by susanev** [comment]
missed these...
- [ ] client run node detail diff has some color contrast issues

**Comment by susanev** [comment]
- [ ] need to fix the color contrast on client run top checkbox

File: components/automate-ui/src/app/page-components/deletable-node-control/deletable-node-control.component.html [hunk 1]
```diff
@@ -2,7 +2,7 @@
   label="Delete checked nodes"
   (click)="showConfModal()"
   [disabled]="selectedNodesIsEmpty()">
-    Delete {{nodesSelectedForDelete.size }} node{{ nodesSelectedForDelete.size | i18nPlural: pluralMapping }}
+    Delete {{nodesSelectedForDelete.size }} Node{{ nodesSelectedForDelete.size | i18nPlural: pluralMapping }}
```
Comment (susanev) [inline components/automate-ui/src/app/page-components/deletable-node-control/deletable-node-control.component.html:5]: bonus.

**Comment by jonong1972** [comment]
Looks good to me 👍

File: components/automate-ui/src/app/pages/+compliance/+reporting/+reporting-overview/overview-status/overview-status.component.html [hunk 2]
```diff
@@ -47,7 +47,7 @@
   </dd>
   <dt class="legend-item-total">{{ data.waived | number }}</dt>
   <dd class="legend-item-label">
-    <span class="legend-item-status skipped"></span>
+    <span class="legend-item-status waived"></span>
```
Comment (susanev) [inline components/automate-ui/src/app/pages/+compliance/+reporting/+reporting-overview/overview-status/overview-status.component.html:50]: this was wrong, i fixed it.

File: components/automate-ui/src/styles/_colors.scss [hunk 3]
```diff
@@ -51,16 +51,17 @@ $blue: #174AF0;
 $malibu: #648FFF;
 // secondary
 $pink: #DC267F;
-$violet: #C22DD5;
+$violet: #A424B4;
 $indigo: #785EF0;
 $science-blue: #0075DB;
+$medium-teal-blue: #005BAB;
```
Comment (susanev) [inline components/automate-ui/src/styles/_colors.scss:57]: this was in the ui library but not here, so i added it.

Comment (msorens) [reply components/automate-ui/src/styles/_colors.scss:57]: Not sure this is appropriate and/or it is a bigger bucket of fish to fry. We are importing the styles from the chef-ui-library here:
https://github.com/chef/automate/blob/4ed76448b0d9e9298b2c0b6690eded29f9830341/components/automate-ui/src/styles.scss#L5
So what you have done is technically redundant. So maybe we do not want/need to import styles from chef-ui-library. But then there are probably other colors that have been duplicated this way, too. 🤷 No idea what the right answer is here.

**Comment by apriofrost** [comment]
Looks good to me! 👍

**Pull request merged**